docs/contribute/contribute.adoc

```diff
@@ -135,40 +178,53 @@ After the review is done and one or more experienced developers is complaining
 about your code you have to do some changes. There are two ways to change your
 code in a submitted pull request:
 
- . Add a new commit or,
- . edit existing commits.
+ . Add a new commit,
+ . or edit existing commits.
 
 ==== Add a new commit
-Adding a new commit is easy and I would choose this option if I would have to
-add something new like a function or a new module.
+Adding a new commit is easy and it is a good option if you have to add something
+new like a function or a new module.
 
 ==== Edit existing commits
-If you just need to fix a typo or add a condition I would choose to go back to
-the commit, where the change is needed, and use commit's `--ammed` option to
-change the commit. You can use following steps to do that.
+If you just need to fix something (for example a typo) you need to go back to
+the commit where the change is needed and use commit's `--ammed` option to
+change the commit. You can use the following steps to do that:
 
 [[app-listing]]
 [source,bash]
 ----
 # show all the commits in your fix_455 branch
-$ git rebase -i maint-1.0
-# replace 'pick' with 'e' at a line with commit you'd like to change
+$ git rebase -i maint-1.2
+# replace 'pick' with 'e' at the line with commit(s) you'd like to edit
 # make your changes
 # vim my_source_file.c
 # commit your new changes
 $ git commit --amend
-# get back to your last commit
+# move to the next commit which you selected for editing using 'e' in the
+# 'git rebase' command
 $ git rebase --continue
 ----
 
+When you are finished with editing commits you can force push all the changes
+into your remote repository to update it with your latest edits. The pull
+request will be updated automatically too:
+
+[[app-listing]]
+[source,bash]
+----
+$ git push --force origin fix_455
+----
+
 === Closing the pull request
 Once the pull request has been merged to upstream's branch the pull request will
 be closed automatically. The issue will be also closed if you used the right
-keyword in the commit message. Now you can delete your `fix_455` branch.
+keyword in the commit message. Now you can delete your `fix_455` branch:
 
 [[app-listing]]
 [source,bash]
 ----
-# detele the fix_455 branch
+# detele the fix_455 branch locally
 $ git branch -d fix_455
+# optionally also delete the fix_455 branch from your remote forked repository
+$ git push origin --delete fix_455
```

Comment:
or you can just click the github button under the PR

Reply:
I think this way is better as it will reflect in both, your cloned repo and also your remote. If you just click the button it will remove a branch from your forked repo on GH but the reference to it will remain in your cloned repo and you will need to `git remote prune origin` to sync it.